vello_shaders/src/compile/mod.rs

```diff
@@ -198,3 +204,11 @@ fn postprocess(wgsl: &str) -> String {
     }
     output
 }
+
+/// Returns the absolute path to the directory containing the WGSL shaders.
+///
+/// The path is determined at compile time.
+pub fn shader_dir() -> &'static PathBuf {
+    static SHADER_DIR: OnceLock<PathBuf> = OnceLock::new();
+    SHADER_DIR.get_or_init(|| Path::new(env!("CARGO_MANIFEST_DIR")).join("shader"))
```

Comment:
If we used `concat!`, I don't think the runtime string manipulation would be needed.

I guess this is about using the proper separator?

Reply:
Just using `concat!` is buggy because we don't know if `CARGO_MANIFEST_DIR` ends with a separator or not. Instead of doing some sort of `.ends_with` dance and potentially missing some other edge cases, better to use the `path` module.